src/Calculator/Views/UnitConverter.xaml

```diff
@@ -615,7 +618,10 @@
                                          x:Uid="RefreshButtonText"
                                          Foreground="{ThemeResource SystemControlHyperlinkBaseHighBrush}"
                                          Click="CurrencyRefreshButton_Click"/>
-                        <TextBlock Margin="0,7,0,0" Style="{ThemeResource CaptionTextBlockStyle}">
+                        <TextBlock x:Name="CurrencySecondaryStatusTextBlock"
+                                   Margin="0,7,0,0"
```

Comment:
Because the control will be collapsed, we are losing the margin (7 pixels vertically) between the refresh button and the bottom of the window (or the numpad)

Reply:
(this margin should be removed and `Margin="0,0,0,7"` should be added to CurrencyRefreshBlockControl)

Reply:
My bad, it was a horizontal stackpanel!